tokens/shitzu.json

```diff
@@ -4,6 +4,6 @@
     "name": "Shitzu",
```

Comment:
Can you please add the `"near_address":`

Reply:
done, I also changed aurora_address. The old address is from post migration. In the meantime we migrated to Near without Rainbow Bridge, which resulted in a new token contract, which has a different address on Aurora